### PR TITLE
Add Arguments Guide to the README file

### PR DESCRIPTION
**Pull request by aaryanrr**
Closes: #51 

#### Describe the changes you've made
1. The Documentation for the arguments that can be passed on to the CLI has been added to the README file
2. Some Typos in the README were fixed
3. Added the `alt` parameter to the `` tag in the CONTRIBUTORS section of the README

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

Updated:
1. Documentation in the README

image

2. `alt` parameter in the `` tag

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,13 +89,13 @@ git push
 **9.** **Congratulations!** Sit and relax, you've made your contribution to Dynamic-CLI project.
 
 ## Contributors:
-### Credits goes to these peoples:✨
+### Credit goes to these peoples:✨
```
Comment (anshsrtv) [inline README.md:92]:
```suggestion
### Credit goes to these people:✨
```

**Review by anshsrtv — CHANGES_REQUESTED** [review]
Nice work 🎉  But a better way of doing the same has been given in the linked issue. Please consider it for your reference.

> ## Usage
> ```
> Usage: Dynamic [OPTIONS]
> 
> A Modern, user-friendly command-line HTTP client for the API testing, and if you're stuck - Search and browse StackOverflow without leaving the CLI.
> 
> Options:
>   -v, --version                 gives the version of our CLI
>   -s, --search                  Search a question on StackOverflow
>  -d,  --debug                   Turn on debugging mode
>   --help                        Show this message and exit.
> ```

**Comment by aaryanrr** [comment]
@anshsrtv I'll do it according to that. Thanks for reviewing.

**Comment by aaryanrr** [comment]
@anshsrtv Please have a look at this now!

**Pull request merged**